src/Resources/app/administration/src/module/findologic-module/page/findologic-page/index.js

```diff
@@ -56,7 +56,13 @@ Component.register('findologic-page', {
             this.shopkeyErrorState = null;
             if (this.isValidShopkey) {
                 this._isStagingRequest();
+            } else {
```

Comment:
```js
// ...
else {
    if {
       // ...
    }
}
```

Also known as `elseif:

```suggestion
            } else if (/** ... */) {
```